src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,35 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.getSessionFactory().openSession();
```

Comment:
In accordance with the checklist, you don't need to call `getSessionFactory()` on the `factory` object because `factory` is already an instance of `SessionFactory`. You should directly call `openSession()` on `factory`.